skyportal/api_description.md

```diff
@@ -8,6 +8,9 @@ inside of `.tokens.yaml`.
 
 Once you have a token, you may access SkyPortal programmatically as
 follows.
+Any JSON request body parameters (all parameters listed under
```

Comment:
Can we remove this paragraph so that we can get to the first example.

I did that in my PR, so not sure why it didn't disappear.